Example/TABTestKit.xcodeproj/project.pbxproj

```diff
@@ -649,6 +649,7 @@
 				CLANG_ANALYZER_NUMBER_OBJECT_CONVERSION = YES_AGGRESSIVE;
 				CLANG_ENABLE_MODULES = YES;
 				CLANG_WARN_DOCUMENTATION_COMMENTS = YES;
+				"CODE_SIGN_IDENTITY[sdk=macosx*]" = "Apple Development";
```

Comment:
This might the reason the builds are failing on Travis. Is there a reason you added this in?